fixtures/cops.py

```diff
@@ -0,0 +1,82 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+import pytest
+import requests
+
+import tldextract
+from rex_redirects import get_rex_release_json_url
+
+from tests.utils import gen_from_file, skip_if_destructive_and_sensitive
+
+DATA_DIR = os.path.join(os.path.realpath(os.path.dirname(__file__)), "data", "webview")
```

Comment:
Since you're splitting out the fixtures anyways (which makes sense), should the data files `s3_books_uuids.txt` and `s3_books_titles.txt` perhaps live in `data/cops` instead of `data/webview`?

Reply:
Good point. Didn't think of it.